src/pages/MainHome.jsx

```diff
@@ -90,14 +90,15 @@ const MainHome = () => {
   };
 
   const filteredData = data?.results.filter((item) => {
```

Comment:
what about like this 

`const filteredData = data?.results.filter(({ email = "", name = "" }) => {
  return email.toLowerCase().includes(lowercasedSearch) || name.toLowerCase().includes(lowercasedSearch);
});`